### PR TITLE
Changes for staging

### PR DESCRIPTION
**Pull request by rfrabasile**
Added Login option
Step recording and retrieving
US country for signup and signin (testing)
SEO meta tags
Other staging scenario changes (hide covid links, etc)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -134,7 +137,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Covid Recovery Corps</title>
+    <title>MindkKind</title>
```
Comment (geoffroth) [inline public/index.html]: Typo

File: src/components/form/types.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 export const COUNTRIES = {
   UK: 0,
-  INDIA: 1,
-  SOUTH_AFRICA: 2,
+  IN: 1,
+  ZA: 2,
   OTHER: 3,
+  US: 4,
 }
```
Comment (geoffroth) [inline src/components/form/types.js:7]: Are we storing the country in attributes? If so, do we want to ensure we're storing a format that is human readable?

File: src/components/widgets/TopNav.tsx [hunk 3]
```diff
@@ -351,15 +362,15 @@ export const TopNav: React.FunctionComponent<TopNavProps> = props => {
...
+      {/*props.token && sessionData.consented && (
         <NavLink
           to="/settings"
           className={classes.fullNavBarLink}
           activeClassName={classes.fullNavBarLinkActive}
         >
           {t('topnav.text7')}
         </NavLink>
-      )}
+      )*/}
```
Comment (geoffroth) [inline src/components/widgets/TopNav.tsx]: Avoid moving commented code up to staging where possible

File: src/helpers/RandomFlowGenerator.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 export const FLOW_OPTIONS = {
-  ONE: 'test_arm_1',
-  TWO: 'test_arm_2',
-  THREE: 'test_arm_3',
-  FOUR: 'test_arm_4',
+  ONE: 'researcher_norms',
+  TWO: 'youth_informed',
+  THREE: 'hybrid',
+  FOUR: 'participant_choice',
```
Comment (geoffroth) [inline src/helpers/RandomFlowGenerator.tsx:5]: 👍

**Review by geoffroth — COMMENTED** [review]
Just one or two minor tweaks.

File: src/components/dashboard/commonConsentSections/FirstCommonConsentSection.tsx [hunk 5]
```diff
@@ -152,6 +156,11 @@ function FirstCommonConsentSection({
                     'This study will only require you to answer weekly survey questions.',
                   )
                   setErrorMessage('')
+                  updateClientData(
+                    step,
+                    'howToParticipate',
```
Comment (carlosalvarado1988) [inline src/components/dashboard/commonConsentSections/FirstCommonConsentSection.tsx]: Could we add a constants file, so we centralize all keys - values if we need to adjust in the near future.

File: src/components/form/types.js (hunk 2 above)
Comment (rfrabasile) [reply src/components/form/types.js:7]: not yet, maybe store the country code like US, UK, IN, ZA

File: src/components/dashboard/commonConsentSections/FirstCommonConsentSection.tsx (hunk 5 above)
Comment (rfrabasile) [reply src/components/dashboard/commonConsentSections/FirstCommonConsentSection.tsx]: indeed

File: src/App.tsx [hunk 6]
```diff
@@ -363,174 +365,184 @@ function App() {
...
-                />
-                <TopNav
-                  token={token}
-                  logoutCallbackFn={() =>
-                    setUserSession(undefined, '', false, [])
-                  }
-                  showTopNavigator={currentLocation !== '/testkit'}
-                >
-                  {/* A <Switch> looks through its children <Route>s and
+        <ElegibilityProvider>
```
Comment (rfrabasile) [inline src/App.tsx:368]: Just added the provider here

File: src/components/form/types.js (hunk 2 above)
Comment (carlosalvarado1988) [reply src/components/form/types.js:7]: Lets goo with  US, UK, IN, ZA for now, I'll communicate Sonia they should look for these codes, unless they need different.

**Pull request merged**